mysql-test/suite/tianmu/r/out_of_range_issue1151.result

```diff
@@ -22,10 +22,6 @@ insert into tiny values(128, 0);
 ERROR 22003: Out of range value for column 'a' at row 1
 insert into tiny values(1234, 0);
 ERROR 22003: Out of range value for column 'a' at row 1
-insert into tiny values(0, 128);
```

Comment:
Why deleted?

Reply:
Temporary, as soon as this submission is merged

Reply:
Temporary, as soon as this submission is merged